apps/workspace-host/src/interface.js

```diff
@@ -104,8 +105,23 @@ let workspace_server_settings = {};
 async
   .series([
     async () => {
-      const configFilename = argv['config'] ?? 'config.json';
-      await loadConfig(configFilename);
+      // For backwards compatibility, we'll default to trying to load config
+      // files from both the application and repository root.
+      //
+      // We'll put the app config file second so that it can override anything
+      // in the repository root config file.
+      let configPaths = [
+        path.join(REPOSITORY_ROOT_PATH, 'config.json'),
+        path.join(APP_ROOT_PATH, 'config.json'),
+      ];
+
+      // If a config file was specified on the command line, we'll use that
+      // instead of the default locations.
+      if ('config' in argv) {
+        configPaths = [argv['config']];
+      }
+
+      await loadConfig(configPaths);
```

Comment:
This same code exists in PrairieLearn; it supports the (legacy, I suppose) pattern of having a single `config.json` in the root of the repo that is ready by both `prairielearn` and `grader-host`.